### ISSUE TITLE
Find or create an interactive app for doing LE2

### ISSUE THREAD

**Issue by StevenClontz** [TeamBasedInquiryLearning/library#21]
Okay if there's one thing I'm okay with a computer helping students with, it's practicing and demonstrating understanding of how to find an RREF, rather than writing these steps out where it's too easy for both the student and instructor to miss subtle errors.

I still want to maintain the CheckIt exercise for this, but an alternative practice/assessment that's more interactive would be quite helpful for all parties involved here I think.

**Comment by siwelwerd** [comment]
Or do we instead need to work to control the  difficulty imbalances in variants of the relevant CheckIt?

**Comment by StevenClontz** [comment]
I think that's a fine stopgap.

**Comment by StevenClontz** [comment]
Less pithy response now that I'm less in a rush: finding a clever way to produce interesting and more consistent matrices is a great improvement to status quo. But I still think that this particular task is actually one of the situations where the student doesn't benefit as much from me reviewing their steps post-facto, compared with a theoretical app to guide them through practicing and demonstrating competency in the skill.

**Comment by siwelwerd** [comment]
I tend to agree, but mostly am wondering how many instructors in general would agree and want to go this way versus a better functioning (i.e. perceived as more "fair") version of what we already have.

**Comment by StevenClontz** [comment]
Ah, sure. I don't think we should remove it from the bank in any situation, so it's worthwhile to improve the matrix generation. I'll make a separate issue to track that.

**Comment by jford1906** [comment]
I think this is what you're after. I can ask permission from the author to recreate it. I bet I could get it to work as a frame in PreText, if we think that would be useful.
https://textbooks.math.gatech.edu/ila/demos/rrinter.html
